### DISCUSSION TITLE
Checking if game is running on SteamDeck

### DISCUSSION THREAD

**Discussion by SolyomBenjamin** [Facepunch/Facepunch.Steamworks#712] [Q&A]
Hello! 
May I ask for a little help?

I've been trying to use the latest Facepunch Steamworks with the latest Steam SDK in Unity to check in runtime if the game is running on SteamDeck or not. As I understand there should be a function for that-> Steamworks.SteamUtils.IsSteamRunningOnSteamDeck();. But I can't manage to make it appear in the available function list no matter what I do. I tried simply with updateing the latest Facepunch steamworks in my project with the included steam sdk stuff -> didn't work. And I tried using the latest steam SDK directly from the steam website -> did'nt work either. What am I doing wrong?

Has anyone had the same issue? Or could anyone manage to use this function in their Unity project? If so, could you please enlighten me about it? :)

Thank you!

**Comment by vincent-savysoda** [comment]
This plugin doesn't have it because its not utilizing the latest steam SDKs. Its the main reason why I've switched to Steamworks.NET currently.

**Comment by TheRealDorcan** [comment]
Same issue. However the code source was updated to handle Steam sdk 1.55!
I'm having trouble compiling it however. Anyone had any luck with it ?

**Reply by TheRealDorcan** [reply]
Well I managed to build the code source, so now I can access SteamUtils.IsRunningOnSteamDeck in Unity :D

**Reply by AlSqPg** [reply]
Any idea how I can do this...?
